html2asketch/model/base.js

```diff
@@ -7,11 +7,17 @@ class Base {
     this._style = null;
     this._objectID = generateID();
     this._name = '';
-    this._resizingConstant = 63;
+    this.setResizingConstraints('none');
   }
 
   setFixedWidthAndHeight() {
-    this._resizingConstant = 12;
+    this.setResizingConstraints('width', 'height');
```

Comment:
I'm not sure if people ever used this method but 12 isn't a valid value for `resizingContraint`

Reply:
huh, good catch! No one complained, so I guess no one used it 😉

Reply:
BTW I like the API, but we should use constants instead of strings (e.g. `this.setResizingConstraints(resizingConstraintValues.width, resizingConstraintValues.height)`) to make sure we avoid any typos